### PR TITLE
Update shared-pkgs tutorial

### LINKED ISSUE #985: 'Install Shared Packages' page issue
From URL: https://www.dartlang.org/tutorials/libraries/shared-pkgs

Under step 3 of https://www.dartlang.org/tutorials/libraries/shared-pkgs#name-the-package-dependencies, there's some misformatted code (just beneath where it says "YAML is picky!"; yes, Markdown can also be picky!). Here's a screenshot:

image

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #985.
Contributes to #692. (Might fix it, depending on how far we want to go.)

Staged at:
https://kw-www-dartlang-1.firebaseapp.com/tutorials/libraries/shared-pkgs

### CODE REVIEW

**Comment by kwalrath** [comment]
I wrote some code for the last section, but it seemed out of scope. Here it is, for posterity:

```dart
import 'package:vector_math/vector_math.dart';

int calculate() {
  var ray = Ray.originDirection(Vector3.zero(), Vector3(1.0, 0.0, 0.0));
  var sphere = Sphere.centerRadius(Vector3(5.0, 0.0, 0.0), 2.0);
  var distanceFromOrigin = ray.intersectsWithSphere(sphere);

  return distanceFromOrigin.round();
}
```

File: src/_tutorials/libraries/shared-pkgs.md [hunk 1]
```diff
@@ -8,22 +8,24 @@ description: Packages are bundles of source code, tools, and resources that help
 <div class="mini-toc" markdown="1">
   <h4>What's the point?</h4>
 
+  * pub.dartlang.org is the primary public repository for Dart packages.
   * Following a few conventions, such as having a valid pubspec.yaml file,
     makes your app a package.
-  * Use Stagehand to generate starting files for your app.
-  * Use `pub get` to download packages.
-  * pub.dartlang.org is the primary public repository for Dart packages.
+  * If you're developing a web or server app,
```
Comment (chalin) [inline src/_tutorials/libraries/shared-pkgs.md]: server -> server-side?

File: src/_tutorials/libraries/shared-pkgs.md [hunk 2]
```diff
@@ -8,22 +8,24 @@ description: Packages are bundles of source code, tools, and resources that help
...
+    use Stagehand to generate starting files.
+  * If you're developing a web or server app,
+    use `pub get` to download packages.
+  * If you're developing a mobile app, use Flutter's tools.
 </div>
 
-Now that you're able to create and run a Dart application
-and have a basic understanding of DOM programming,
-you are ready to leverage code written by other programmers.
+Once you can create and run a Dart application,
```
Comment (chalin) [inline src/_tutorials/libraries/shared-pkgs.md]: application -> app

File: src/_tutorials/libraries/shared-pkgs.md [hunk 3]
```diff
@@ -137,17 +141,16 @@ which is available at pub.dartlang.org.
    YAML is picky!
    For example:
 
-   {% prettify yaml %}
-    environment:
-      sdk: '>=1.20.1 <2.0.0'
+   <pre class="prettyprint lang-yaml">
```
Comment (chalin) [inline src/_tutorials/libraries/shared-pkgs.md]: Hmm, strange that `{% prettify %}` caused formatting problems.

File: src/_tutorials/libraries/shared-pkgs.md [hunk 4]
```diff
@@ -267,7 +268,7 @@ only one, `lib`, was installed when you ran pub get.
 ## Import libraries from a package
 
 Now that you've installed the package,
-you can import its libraries and use them in your Dart file.
+you can import its libraries and use them in your application.
```
Comment (chalin) [inline src/_tutorials/libraries/shared-pkgs.md]: application -> app?

**Comment by chalin** [comment]
> I wrote some code for the last section, but it seemed out of scope.

Agreed. I like the terseness and focus of the current form of the page.

File: src/_tutorials/libraries/shared-pkgs.md (hunk 3 above)
Comment (chalin) [reply src/_tutorials/libraries/shared-pkgs.md]: (I'll investigate this later)

**Comment by kwalrath** [comment]
I fudged the import instructions a bit, hoping that pub.dartlang.org will change their format: https://github.com/dart-lang/pub-dartlang-dart/issues/1430

**Pull request merged**